### PR TITLE
feat: student profile

### PR DESCRIPTION
**Pull request by 01001110J**
# Description

Here is added the student profile view.

This PR resolves [PADV-1635](https://agile-jira.pearson.com/browse/PADV-1635)

## Change log
- Created new view for student profile.
- Updated some columns to add link for student profile.
- Updated paths in router for student profile.
- Added error message component for edge cases.
- Updated unit test.

### Visual results
dd
afte

### How to test
- Go to `/institution-portal/students` and pick any student.

### CODE REVIEW

File: src/features/Students/StudentDetailPage/index.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.nav-tabs a {
+    text-decoration: none;
+}
```
Comment (AuraAlba) [inline src/features/Students/StudentDetailPage/index.scss:3]: Add the empty line in the end

**Review by AuraAlba — CHANGES_REQUESTED** [review]
